AP-protocol.md

```diff
@@ -307,30 +307,42 @@ TODO
 
 # Simulation Controller
 
-Like nodes, the **Simulation Controller** runs on a thread. It must retain a means of communication with all nodes of the network, even when drones go down.
+Like nodes, the **Simulation Controller** (SC) runs on a thread. It must retain a means of communication with all nodes of the network, even when drones go down. 
+The Simulation controller can send and receive different commands to/from the nodes (drones, clients and servers) through reserved channels. The list of available **commands** is as follows:
 
-### Simulation commands
-
-The Simulation Controller can send the following commands to drones:
-
-`Crash`: This commands makes a drone crash. Upon receiving this command, the drone’s thread should return as soon as possible.
+```rust
+pub enum Command {
+    AddChannel(NodeId, Sender<Packet>),
+    RemoveChannel(NodeId),
+    Crash,
+    ChangePdr(f32),
+    SentMessage(Packet),
+    DroppedMessage(Packet)
+}
+```
 
-`AddSender(crossbeam::Sender, dst_id)`: This command provides a node with a new crossbeam Sender to send messages to node `dst_id`.
+### Simulation commands
 
-`AddReceiver(mpsc::Receiver, src_id)`: This command provides a node with a new crossbeam Receiver to receive messages from node `src_id`.
+Below are documented the commands that can be exchanged between the **SC and any type of node** (drone, client, server):
 
-`Spawn(id, code)`: This command adds a new drone to the network.
+- **SC -> Node (any)**
 
-`SetPacketDropRate(id, new_pdr)`:
+  - `Crash`: This commands makes a node crash. Upon receiving this command, the node’s thread should return as soon as possible. The last thing the node should do is to confirm that it is terminating by sending a Crash command to the SC. 
+  - `AddChannel(NodeId, Sender<Packet>)`: This command tells a node to add a new **edge** to the node with id `NodeId` and to use the provided `Sender<Packet>` channel to communicate with it. Since we are assuming bidirectional communication, this command must be used **two times** (i.e. to add the edge from `A` to `B`, we must add the `Sender from A to B` and the `Sender from B to A`).
```

Comment:
That PR is about adding a way to store the different Sender channels in a drone. When a drone receives the AddChannel command, it will add that Sender to the HashMap. Or is the drone initialized with all the available Senders (I hope it is not intended this way)?

Reply:
Im closing these two conversations, leaving open only the one about splitting the enum.